compiler/il/OMRIL.cpp

```diff
@@ -836,6 +678,22 @@ OMR::IL::opCodeForIndirectArrayStore(TR::DataType dt)
 TR::ILOpCodes
 OMR::IL::opCodeForRegisterLoad(TR::DataType dt)
    {
+   static_assert(TR::NumOMRTypes == (sizeof(OMR::IL::opCodesForRegisterLoad) / sizeof(OMR::IL::opCodesForRegisterLoad[0])),
+              "OMR::IL::opCodesForRegisterLoad is not the correct size");
+
+   if (dt.isVector())
+      {
+      switch (dt.getVectorElementType())
+         {
+         case TR::Int8:   return TR::vbRegLoad;
```

Comment:
Why do we have all these different "vector register load" opcodes when we only have a single equivalent "vector load" opcode ?

Reply:
Actually, "New Vector Opcodes" PR is coming soon. It introduces new infrastructure for vector opcodes. Then, we will gradually remove old opcodes and add new ones.